action.yml

```diff
@@ -1,9 +1,9 @@
-name: rdme
+name: rdme 🦉 Sync to ReadMe
 author: ReadMe
 branding:
   color: blue
   icon: book-open
-description: Update your ReadMe developer hubs in a CI environment.
+description: Sync OpenAPI (OAS) definitions and Markdown documentation from your GitHub repository to your ReadMe developer hub
```

Comment:
no punctuation?

Reply:
That seems to be the pattern most other listings follow 🤷🏽  https://github.com/marketplace?type=actions